roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -83,28 +82,55 @@
         '{{ etcd_cert_dir }}/member-{{ node }}.pem',
         '{{ etcd_cert_dir }}/member-{{ node }}-key.pem',
         {% endfor %}]"
+  delegate_to: "{{ groups['etcd'][0] }}"
+  run_once: yes
+  when:
+    - inventory_hostname in groups['etcd']
+    - sync_certs|default(false)
+    - inventory_hostname != groups['etcd'][0]
```

Comment:
oops - CI failed
It seems that in the presence of `run_once`, first the host (`instance-1` in CI ) on which the task is played is determined, and then it is skipped, since it does not satisfy the when condition; (

so
please remove `run_once: yes`
sorry for my mistake

Reply:
No problem. run_once removed